.github/workflows/ci_cd.yml

```
@@ -102,7 +102,7 @@ jobs:
   docs:
     name: Documentation
     if: github.event.action != 'closed'
-    runs-on: ubuntu-latest
+    runs-on: public-ubuntu-latest-8-cores
```

We also have the 16 cores version

```suggestion
    runs-on: public-ubuntu-latest-16-cores
```